src/main/scripts/src/app/workspace/organization/organization-settings.component.html

```diff
@@ -1,14 +1,71 @@
-<ul class="nav nav-tabs">
-  <li role="presentation" class="nav-item">
-    <a routerLink="settings" routerLinkActive="active" class="nav-link">
-      Basic Settings
-    </a>
-  </li>
-  <li role="presentation" class="nav-item">
+<div class="container-fluid">
+  <div class="row">
+    <div class="col"></div>
+    <div class="whole col-sm-9 col-md-10 col-lg-11" [style.opacity]="initialized() ? '1' : '0'">
```

Comment:
It behaves very strangely now. The larger the window is, the smaller the gaps are. It should be exactly the opposite. Try to use `col-12 col-lg-11 col-xl-10` as I recommended.

Permissions table and tabs above it should also use this layout. It does not look good at all now.